### ISSUE TITLE
Server Crash (Multiple entries with same key: Splash Potion of Levitation)

### ISSUE THREAD

**Issue by CapoFantasma97** [cc-tweaked/cc-restitched#48]
### Minecraft Version

1.17.x

### Version

1.97.3

### Details

The latest 1.17.1 patch, 1.97.3, makes our modded private server crash. No Crashes occur with 1.97.2, and the client is fine with either version.

Logs points out to multiple entries with the same key (item.minecraft.splash_potion.effect.levitation) could it be the culprit?

```
[12:55:13] [main/ERROR]: Minecraft has crashed!
net.fabricmc.loader.impl.FormattedException: java.lang.ExceptionInInitializerError
	at Not Enough Crashes deobfuscated stack trace.(1.17.1+build.65) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:610) ~[fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotServer.main(KnotServer.java:23) [fabric-loader-0.12.12.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:66) [fabric-loader-0.12.12.jar:?]
Caused by: java.lang.ExceptionInInitializerError
	at net.minecraft.text.TranslatableText.updateTranslations(TranslatableText:42) ~[?:?]
	at net.minecraft.text.TranslatableText.visitSelf(TranslatableText:143) ~[?:?]
	at net.minecraft.text.Text.visit(Text:96) ~[?:?]
	at net.minecraft.text.StringVisitable.getString(StringVisitable:91) ~[?:?]
	at net.minecraft.text.Text.getString(Text:39) ~[?:?]
	at com.mojang.brigadier.exceptions.CommandSyntaxException.<init>(CommandSyntaxException.java:27) ~[server-intermediary.jar:?]
	at com.mojang.brigadier.exceptions.SimpleCommandExceptionType.createWithContext(SimpleCommandExceptionType.java:21) ~[server-intermediary.jar:?]
	at com.mojang.brigadier.StringReader.readDouble(StringReader.java:142) ~[server-intermediary.jar:?]
	at net.minecraft.command.argument.CoordinateArgument.parse(CoordinateArgument:41) ~[?:?]
	at net.minecraft.command.argument.DefaultPosArgument.parse(DefaultPosArgument:87) ~[?:?]
	at net.minecraft.command.argument.Vec3ArgumentType.parse(Vec3ArgumentType:54) ~[?:?]
	at net.minecraft.command.argument.Vec3ArgumentType.parse(Vec3ArgumentType:21) ~[?:?]
	at com.mojang.brigadier.tree.ArgumentCommandNode.isValidInput(ArgumentCommandNode.java:91) ~[server-intermediary.jar:?]
	at com.mojang.brigadier.tree.CommandNode.findAmbiguities(CommandNode.java:102) ~[server-intermediary.jar:?]
	at com.mojang.brigadier.tree.CommandNode.findAmbiguities(CommandNode.java:113) ~[server-intermediary.jar:?]
	at com.mojang.brigadier.tree.CommandNode.findAmbiguities(CommandNode.java:113) ~[server-intermediary.jar:?]
	at com.mojang.brigadier.CommandDispatcher.findAmbiguities(CommandDispatcher.java:695) ~[server-intermediary.jar:?]
	at net.minecraft.server.command.CommandManager.<init>(CommandManager:235) ~[?:?]
	at net.minecraft.resource.ServerResourceManager.<init>(ServerResourceManager:39) ~[?:?]
	at net.minecraft.resource.ServerResourceManager.reload(ServerResourceManager:88) ~[?:?]
	at net.minecraft.server.Main.main(Main:147) ~[server-intermediary.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608) ~[fabric-loader-0.12.12.jar:?]
	... 7 more
Caused by: java.lang.IllegalArgumentException: Multiple entries with same key: item.minecraft.splash_potion.effect.levitation=Splash Potion of Levitation and item.minecraft.splash_potion.effect.levitation=Splash Potion of Levitation
	at com.google.common.collect.ImmutableMap.checkNoConflict(ImmutableMap.java:186) ~[server-intermediary.jar:?]
	at com.google.common.collect.RegularImmutableMap.checkNoConflictInKeyBucket(RegularImmutableMap.java:102) ~[server-intermediary.jar:?]
	at com.google.common.collect.RegularImmutableMap.fromEntryArray(RegularImmutableMap.java:88) ~[server-intermediary.jar:?]
	at com.google.common.collect.ImmutableMap$Builder.build(ImmutableMap.java:359) ~[server-intermediary.jar:?]
	at net.minecraft.util.Language.create(Language:45) ~[?:?]
	at net.minecraft.util.Language.<clinit>(Language:33) ~[?:?]
	at net.minecraft.text.TranslatableText.updateTranslations(TranslatableText:42) ~[?:?]
	at net.minecraft.text.TranslatableText.visitSelf(TranslatableText:143) ~[?:?]
	at net.minecraft.text.Text.visit(Text:96) ~[?:?]
	at net.minecraft.text.StringVisitable.getString(StringVisitable:91) ~[?:?]
	at net.minecraft.text.Text.getString(Text:39) ~[?:?]
	at com.mojang.brigadier.exceptions.CommandSyntaxException.<init>(CommandSyntaxException.java:27) ~[server-intermediary.jar:?]
	at com.mojang.brigadier.exceptions.SimpleCommandExceptionType.createWithContext(SimpleCommandExceptionType.java:21) ~[server-intermediary.jar:?]
	at com.mojang.brigadier.StringReader.readDouble(StringReader.java:142) ~[server-intermediary.jar:?]
	at net.minecraft.command.argument.CoordinateArgument.parse(CoordinateArgument:41) ~[?:?]
	at net.minecraft.command.argument.DefaultPosArgument.parse(DefaultPosArgument:87) ~[?:?]
	at net.minecraft.command.argument.Vec3ArgumentType.parse(Vec3ArgumentType:54) ~[?:?]
	at net.minecraft.command.argument.Vec3ArgumentType.parse(Vec3ArgumentType:21) ~[?:?]
	at com.mojang.brigadier.tree.ArgumentCommandNode.isValidInput(ArgumentCommandNode.java:91) ~[server-intermediary.jar:?]
	at com.mojang.brigadier.tree.CommandNode.findAmbiguities(CommandNode.java:102) ~[server-intermediary.jar:?]
	at com.mojang.brigadier.tree.CommandNode.findAmbiguities(CommandNode.java:113) ~[server-intermediary.jar:?]
	at com.mojang.brigadier.tree.CommandNode.findAmbiguities(CommandNode.java:113) ~[server-intermediary.jar:?]
	at com.mojang.brigadier.CommandDispatcher.findAmbiguities(CommandDispatcher.java:695) ~[server-intermediary.jar:?]
	at net.minecraft.server.command.CommandManager.<init>(CommandManager:235) ~[?:?]
	at net.minecraft.resource.ServerResourceManager.<init>(ServerResourceManager:39) ~[?:?]
	at net.minecraft.resource.ServerResourceManager.reload(ServerResourceManager:88) ~[?:?]
	at net.minecraft.server.Main.main(Main:147) ~[server-intermediary.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608) ~[fabric-loader-0.12.12.jar:?]
	... 7 more

**Comment by Merith-TK** [comment]
That makes me think that someone splashed a turtle as it was moving... But something tells me that's wrong as well.. Or a turtle did something with a splash potion??

**Comment by toad-dev** [comment]
This is a duplicate of #39. The fix will be included in the next 1.17 release. You can grab the latest actions build from the 1.17.x branch if you want to test it early.

**Comment by toad-dev** [comment]
[The latest release](https://github.com/cc-tweaked/cc-restitched/releases/tag/v1.17.1-1.97.4-ccr) fixes this issue. Please update if you can!
